0.87/demos/st_metric.py

```diff
@@ -10,22 +12,75 @@ def show():
         """
         # Try out `st.metric`!
 
-        It lets you to display a number in big bold font in your app. Really useful 
-        for KPIs or other important metrics.
+        Display KPIs or important metrics in big bold font in your app
+        ---
         """
     )
 
-    st.metric("Streamlit version", 0.87, "0.01")
+    st.subheader('API')
+    st.write("Here's an example which shows the API and `st.metric` in action")
+    with st.echo():
+        st.metric(label="Streamlit version", value=0.87, delta="0.01")
```

Comment:
```suggestion
        st.metric(label="Streamlit version", value=0.87, delta=0.01)
```

(I think I already had string for delta here in the original app, but let's change it to float to make it more consistent for the user.)

Reply:
Fixed